addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16412,7 +16412,7 @@ msgstr ""
 #. Description of setting #14102 Settings -> Video -> Discs -> Blu-ray playback mode
 #: system/settings/settings.xml
 msgctxt "#37031"
-msgid "Specifies how Blu-rays should be opened/played back. Disc menus are not fully supported yet and may cause problems."
+msgid "Specifies how Blu-ray's should be opened/played back. Disc menus are not fully supported yet and may cause problems."
```

Comment:
I'm not a native English speaker but I'm 100% certain that it's `Blu-rays`. The `'s` must only be used in a possessive context and this here isn't. If you want to talk about a track on a Blu-ray you can write `Blu-ray's track` and if you talk about tracks on multiple Blu-rays you write `Blu-rays' track` but as a simple plural it is `Blu-rays`.